releasenotes/notes/use-cloud-provider-ntp-2995657d00d3007d.yaml

```
@@ -0,0 +1,4 @@
+enhancements:
+  - |
+    The NTP check now uses the cloud provider's NTP server by default, and the NTP pool of ntp.org
+    we use if no cloud provider is detected.
```

What do you think about ?

```suggestion
    The NTP check now uses the cloud provider's NTP server by default, if possible.
```